docs/troubleshooting/typed-linting/index.mdx

```diff
@@ -84,18 +84,21 @@ If the IDE provides different type information from typescript-eslint's report,
 
 ## Are TypeScript project references supported?
 
-Yes, but only with [`EXPERIMENTAL_useProjectService`](../../packages/Parser.mdx#experimental_useprojectservice).
+Yes, but only with [`parserOptions.projectService`](../../packages/Parser.mdx#projectservice).
 
 See [issue #2094 discussing project references](https://github.com/typescript-eslint/typescript-eslint/issues/2094) for more details.
 
 ## Project Service Issues
 
-<HiddenHeading id="allowdefaultproject-glob-too-wide" />
-<HiddenHeading id="allowdefaultprojectforfiles-glob-too-wide" />
```

Comment:
This second heading is a holdover from the experimental/v7 name. I think anybody newly configuring the service should reasonably be on v8 now - our docs don't still hold true for the old experimental version.